.gitignore

```diff
@@ -38,6 +38,8 @@ models/model_robertson/build/*
 !models/model_calvetti
 models/model_calvetti/build/*
 
+amici_models/*
```

Comment:
that is the default I used in pypesto. If that leads to problems, one can rename in amici.petab.petab_import

Reply:
Fine for me. Just ignoring, because e.g. gitk otherwise shows hundreds of modified files